<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="UTF-8">
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<title>iDEC NEFU_China 2023</title>
	<link rel="stylesheet" type="text/css" href="../../css/main_cssstyle.css">
	<style>
		header {
			background: url("../../static/top/results.png");
			min-height: 50%;
			background-repeat: no-repeat;
			background-size: cover;
		}

		.page-title {
			padding-top: 10em;
			padding-bottom: 1.5em;
		}

		/* footer的样式 */
		.contact-icon {
			display: flex;
			align-items: center;
			/*font-size: 3em;*/
			color: white;
		}

		.contact-icon a {
			color: white;
		}

		.contact-icon i {
			font-size: 3em;
		}

		.t {
			font-size: 0.8em;
		}
	</style>
</head>

<body>

	<a class="skip-main" href="#slide1-chapter1">Skip to main content</a>
	<header>
		<nav id="normal-nav">
			<!---- 常规导航栏Navigation bar (normal) -->
			<div class="logo">
				<img style="max-height: 22em" id="nav-logo" src="../../static/logo.png"
					alt="This is our logo, but if you click it, you will return home">
			</div>
			<div class="chapters">
				<div class="dropdown-menu cool-link">
					<a class="chapter-nav " href="../../static/home.html">
						<div class="chapter-content">
							<div class="chapter-title active ">
								Home
							</div>
						</div>
					</a>
				</div>
				<div class="dropdown-menu cool-link">
					<a class="chapter-nav " href="#">
						<div class="chapter-content">
							<div class="chapter-title active ">
								Team
							</div>
							<div class="chapter-icon">
								<i class="fas fa-sort-down"></i>
							</div>
						</div>
					</a>
					<div class="dropdown-content">
						<a href="../team/mumbers.html" class="dropdown-page cool-link2">
							Members
						</a>
						<a href="../team/attributions.html" class="dropdown-page cool-link2">
							Attributions
						</a>
					</div>
				</div>
				<div class="dropdown-menu cool-link">
					<a class="chapter-nav " href="#">
						<div class="chapter-content">
							<div class="chapter-title active ">
								Project
							</div>
							<div class="chapter-icon">
								<i class="fas fa-sort-down"></i>
							</div>
						</div>
					</a>
					<div class="dropdown-content">
						<a href="../project/background.html" class="dropdown-page cool-link2">
							Background
						</a>
						<a href="../project/design.html" class="dropdown-page cool-link2">
							Design
						</a>
						<a href="../project/results.html" class="dropdown-page cool-link2">
							Results
						</a>
						<a href="../../static/report.pdf" class="dropdown-page cool-link2" target="_blank">
							Report
						</a>
						<a href="../../static/supplementary information.pdf" class="dropdown-page cool-link2"
							target="_blank">
							Supplementary information
						</a>
					</div>
				</div>
				<div class="dropdown-menu cool-link">
					<a class="chapter-nav " href="#">
						<div class="chapter-content">
							<div class="chapter-title active ">
								Documentation
							</div>
							<div class="chapter-icon">
								<i class="fas fa-sort-down"></i>
							</div>
						</div>
					</a>
					<div class="dropdown-content">
						<a href="../documentation/experiments.html" class="dropdown-page cool-link2">
							Experiments
						</a>
						<a href="../documentation/notebook.html" class="dropdown-page cool-link2">
							Notebook
						</a>
						<a href="../documentation/safety.html" class="dropdown-page cool-link2">
							Safety
						</a>
						<a href="../project/protocol.html" class="dropdown-page cool-link2">
							Protocol
						</a>
					</div>
				</div>
			</div>
		</nav>
		<!------------ 页面标题PAGE TITLE -------------------->
		<div class="page-title">
			<div class="page-title-content">
				<h1>RESULTS</h1>
				<!---Page title -->
			</div>
		</div>
	</header>
	<div class="grid-normal">
		<div class="introduction grid-normal">
		</div>
	</div>
	<div style="display:none" id="marker" class="triangle-down"> </div>
	<!------------------ BUTTONS ---------------------------->
	<div class="buttons">
	</div>
	<div class="sections-container">
		<aside class="sticky">
			<div id="sub-nav" class="sub-navigation-content">
				<div class="sub-navigation-chapter cool-link2">
					<a class="main-nav-link" linkedidchapter="slide1-chapter1">Construction of hmaS mutant library</a>
					<!------Chapter one -->
				</div>
				<div class="sub-navigation-chapter cool-link2">
					<a class="main-nav-link" linkedidchapter="slide1-chapter2">Construction and Test of a Dual-Module
						Screening Platform</a>
					<!------Chapter Two -->
				</div>
				<div class="sub-navigation-chapter cool-link2">
					<a class="main-nav-link" linkedidchapter="slide2-chapter2">
						<li class="t">Functional testing</li>
					</a>
					<!------Chapter Three -->
				</div>
				<div class="sub-navigation-chapter cool-link2">
					<a class="main-nav-link" linkedidchapter="slide2-chapter3">
						<li class="t">Screening to obtain excellent mutants</li>
					</a>
					<!------Chapter Three -->
				</div>
				<div class="sub-navigation-chapter cool-link2">
					<a class="main-nav-link" linkedidchapter="slide3-chapter1">Modeling and Molecular Docking</a>
					<!------Chapter Two -->
				</div>
				<div class="sub-navigation-chapter cool-link2">
					<a class="main-nav-link" linkedidchapter="slide4-chapter1">Engineering an E. coli chassis with
						High-HPP Production</a>
					<!------Chapter Two -->
				</div>
				<div class="sub-navigation-chapter cool-link2">
					<a class="main-nav-link" linkedidchapter="slide4-chapter2">
						<li class="t">Overexpression of Key Genes to Enhance the
							Precursor
							Supply</li>
					</a>
					<!------Chapter Three -->
				</div>
				<div class="sub-navigation-chapter cool-link2">
					<a class="main-nav-link" linkedidchapter="slide4-chapter3">
						<li class="t">Functional testing</li>
					</a>
					<!------Chapter Three -->
				</div>
				<div class="sub-navigation-chapter cool-link2">
					<a class="main-nav-link" linkedidchapter="slide4-chapter4">
						<li class="t">Screening to obtain excellent mutants</li>
					</a>
					<!------Chapter Four -->
				</div>
				<div class="sub-navigation-chapter cool-link2">
					<a class="main-nav-link" linkedidchapter="slide5-chapter1">Evaluation of Metabolic Engineering and
						Directed Evolution</a>
					<!------Chapter Two -->
				</div>

				<!--            返回顶部箭头-->
				<div id="to-top-arrow">
					<a title="Back to top" href="#"><i class="fas fa-chevron-circle-up"></i></a>
					<!------Chapter Three -->
				</div>
			</div>
		</aside>
		<div class="carousel">
			<div class="slider">
				<!------------------------------ section 1------------------------------------>
				<section class="slide-sections">
					<div class="main-body">
						<div class="main-body-content">
							<div id="slide1-chapter1">
								<h2>Construction of hmaS mutant library</h2>
								<p>Based on error-prone PCR, we used a low-fidelity DNA polymerase and adjusted
									Mn<sup>2+ </sup>and
									Mg<sup>2+</sup>>concentrations and ratio to
									manipulate the mutation rate. We determined the final concentrations of 0.02 mM
									Mn<sup>2+</sup>
									and 0.5 mM Mg<sup>2+</sup> for mutagenesis and
									achieved a mutation rate of 2 mutations/kb, producing a high variety of
									mutations.
									We optimized the Golden Gate assembly
									procedure to increase the probability of inserting mutated fragments into
									the
									vector. Eventually, we constructed a
									mutant library containing 1 * 10<sup>5</sup> clones (Fig. 1) and the rate of
									positive clones
									was 100%.</p>
								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/图片1.png" alt="">
									<p>gY9s-HmaS<sup>mut</sup>-Bio177 mutant library plates</p>
								</div>

								<p><b>Figure 1.</b> A mutant library with 10⁵ clones was constructed by using a
									low-fidelity DNA polymerase, determining the concentrations
									of 0.02 mM Mn<sup>2+</sup> and 0.5 mM Mg<sup>2+</sup>and the optimized Golden Gate
									assembly procedure.</p>
							</div>
							<div id="slide1-chapter2">
								<h2>Construction and Test of a Dual-Module Screening Platform</h2>
								<p>
									<b id="slide2-chapter2">Functional testing</b>
								</p>
								<p>We used bacteria harboring the Bio177 plasmid to simulate the scenario of low
									activity, and bacteria containing the
									gY9s-HmaS-Bio177 plasmid for high enzyme activity to test the functionality of
									dual-module screening platform. Both
									strains were spread on selection plates containing 0.01, 0.05 and 0.1 mg/L 5-FC to
									simulate the screening process under
									stress conditions (Fig. 2). We found that, in medium with 0.01 mg/L 5-FC, all 10
									randomly selected clones were verified
									as positive by colony PCR, yielding a 100% selection rate to distinguish desirable
									hmaS-containing clones from others.
									Under condition of 0.05 mg/L 5-FC, the growth of the wild-type HmaS strain was
									inhibited, resulting in a reduced colony
									number on the plate. Ultimately, the WT strain could not grow under 0.1 mg/L 5-FC
									condition.</p>
								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/图片2.png" alt="">
									<p>0.01 mg/L 5-FC</p>
									<p>0.05 mg/L 5-FC 0.1 mg/L 5-FC </p>
								</div>
								<p><b>Figure 2.</b> Spread strains on selection plates containing 0.01, 0.05 and 0.1
									mg/L of 5-FC to simulate the screening process under
									stress conditions.</p>

								<p>
									<b id="slide2-chapter3">Screening to obtain excellent mutants</b>
								</p>
								<p>We transferred the bacterial cultures into the selection media and cultivated the
									bacteria at 30°C until the bacteria
									reached the exponential phase (OD<sub>600</sub> =0.2 or 0.6). With increased 5-FC
									concentrations, the selection pressure
									accelerated, which enhanced the performance of viable individuals, improved HMA
									production, and reduced fluorescence
									strength. Based on colony fluorescence and HMA production, we screened clones under
									the conditions of 0.6 g/L HPP and
									0.3 mg/L 5-FC (Fig. 3). And we found a mutant clone hmaS<sup>V152G</sup> that HMA
									production of
									mutant clones could increase up to
									125% compared to that of WT (Fig. 4).</p>
								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/t10.jpg" alt="">
								</div>
								<p><b>Figure 3.</b> Graph of the correlation between 5-FC concentration in the medium
									and
									HMA production. The orange bars represent HMA
									levels and blue line depicts fluorescence intensity.</p>

								<p>
								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/t8.jpg" alt="">
								</div>
								<p><b>Figure 4.</b> Graph of HMA production by the bacteria harboring
									HmaS<sup>WT</sup>and
									HmaS<sup>V152G</sup>clones. With 9 mM HPP as the substrate for
									whole-cell catalysis, HMA accumulation by the gY9s-HmaS<sup>V152G</sup>-Bio177 and
									gY9s-HmaS<sup>WT</sup>-Bio177 strains was monitored for 3
									h.</p>

								<p>
							</div>
							<div id="slide3-chapter1">
								<h2>Modeling and Molecular Docking</h2>
								<p>We employed the Alphafold2 for homology modeling of both the HmaS<sup>WT</sup>
									and mutant
									HmaS<sup>V152G</sup> (Fig. 5).</p>
								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/图片3.jpg" alt="">
								</div>
								<p><b>Figure 5.</b> The surface models for the spatial structures of the
									HmaS<sup>WT</sup> and
									HmaS<sup>V152G</sup> mutants in the binding domain of HmaS. The
									residue 152 is colored in dark blue. The V152G mutation alters the positions of
									adjacent residues and subsequently
									increases the wideness of the pocket to easily accommodate HPP molecules.</p>
								<p>we used the Autodock Vina to dock the HPP molecule to HmaS proteins. The docking
									energy of HPP with WT and V152G mutant
									was determined as -3.8 and -4.7 kcal/mol, respectively, indicating that HPP is
									likely better oriented towards the HmaS
									mutant. Following docking analysis, we evaluated the ligand-protein interactions
									using Pymol and Ligplus. After HPP
									docking with HmaS<sup>V152G</sup>, HPP forms hydrogen bonds with D154, G199 and
									A204, and
									engages in hydrophobic interactions with
									L153, G203, T206 and R318 (Fig. 6)</p>
								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/图片4.jpg" alt="">
								</div>
								<p><b>Figure 6.</b> Simulated structures of HmaS<sup>V152G</sup>-HPP binding analyzed
									by Pymol
									and Ligplus. Compared to HmaS<sup>WT</sup>
									and mutant, HmaS<sup>V152G</sup> uses G152,
									L154 and E199 to form additional hydrogen bonds with a HPP molecule.</p>

								<p>
							</div>
							<div id="slide4-chapter1">
								<h2>Engineering an E. coli chassis with High-HPP Production</h2>
								<p>
									<b id="slide4-chapter2"> Overexpression of Key Genes to Enhance the
										Precursor
										Supply</b>
								</p>
								<p>we constructed the pSB1c-aroG<sup>fbr</sup>-tktA-ppsA (pATP) plasmid to
									overexpress the
									phosphoenol-pyruvate synthase (<i>ppsA</i>) gene,
									the transketolase (<i>tktA</i>) gene and aro<i>G<sup>fbr</sup></i>gene.
									Through the modifications of
									genetic
									overexpression, we obtained a host
									strain that could produce up to 0.72 g/L HPP.</p>
								<p>
									<b id="slide4-chapter3">Building Multi-level Inhibition Platform Based on
										CRISPRi</b>
								</p>
								<p>
									<strong id="slide4-chapter4">
										Functional testing</strong>
								</p>
								<p>To test the feasibility of multi-level interference, we co-transformed <i>E.
										coli</i>
									BW25113 with R6K-dCas9-eGFP plasmid
									containing the sgRNA targeting the eGFP coding sequence (+40bp) and
									pYB1a-eGFP
									plasmid. We tested the effectiveness of
									sgRNAs with different mismatches in mediating eGFP suppression. The eGFP
									repression
									levels is from 6.2% to 80.3% (Fig.
									7).</p>
								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/图片5.tif" alt="">
								</div>
								<p><b>Figure 7.</b> Bar graph of the Inhibitory efficiency by the random
									double-mismatched eGFP-sgRNAs. Fi means fully interference,
									transformed into the strain that containing the non-mutated R6K-dCas9-eGFP
									plasmid, which had CA in the 7-8bp spacer of
									the sgRNA.</p>
								<p>
								<p>
									<strong id="slide4-chapter5">
										Screening to obtain excellent mutants</strong>
								</p>
								<p>We individually co-transformed the plasmids of these R6K-dCas9-sgRNA single
									or
									four mismatch sgRNA libraries and
									non-mismatch sgRNA with the pSB1c-aroG<sup>fbr</sup>-tktA-pheA plasmid into
									<i>E.
										coli</i>
									BWΔCD,
									induced them in the ZYM5052
									medium, and
									transferred them into M9 medium containing 20 g/L glucose for fermentation.
									After 24 h of cultivation, we
									performed HPLC
									analysis to test their HPP production. Especially, when <i>tyrR</i> was
									silenced, a
									significant increase of HPP was
									achieved
									with an improvement of 113.6% (0.704 g/L) compared to the strains only
									expressing the enhanced upstream
									pathways. We
									selected a non-mutant <i>tyrRi</i> strain for flask fermentation, and the HPP
									titer
									reached 0.91 g/L (Fig. 8).
								</p>
								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/图片7.tif" alt="">
								</div>
								<p><b>Figure 8.</b> Bar graph of HPP production with different genes inhibited by
									CRISPRi. The blue column represents the strain transformed
									by pATP, the green column is for the strain transformed by pATP and
									R6K-dCas9-sgRNA, and the orange columns indicate the
									strain transformed by pATP and R6K-dCas9-sgRNA mutants. The control group was
									the BWΔCD strain carrying no plasmid.</p>
							</div>
							<div id="slide5-chapter1" style="margin-bottom:10%;">
								<h2>Evaluation of Metabolic Engineering and Directed Evolution</h2>
								<p>The optimized clones (gY9s-HmaSV152G-Bio177) obtained from the screening were
									transformed into the bacterial host (pATP
									or tyrRi). The experiment groups were listed in the Table 1. After
									cultivation
									and
									induced protein expression, the
									bacteria were transferred to a fermentation medium using glucose as the
									substrate
									and fermented in a shake flask at 30°C
									for 8 h.</p>
								<p>Table 1. List of plasmids used to generate different strains in each group
									for
									fermentation.</p>
								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/12.jpg" alt="">
								</div>
								<p>Compared HMA1, HMA3 and HMA5 clones, we found that gene overexpression or
									silencing
									alone had limited effects on overall
									fermentation promotion. However, when two approaches are combined, it
									allowed
									the
									endogenous HPP synthesis pathway to
									achieve balanced, stable and sustained expression, lead to a remarkable
									increase
									in
									the fermentation yield.</p>
								<p>Contrasted the HMA1 and HMA2, HMA3 and HMA4, HMA5 and HMA6 pairs, we found
									that
									the
									essential limiting factor was the
									enzymatic activity of hydroxymandelate synthase, which significantly
									restricted
									HMA
									production (Fig. 9).</p>

								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/11.jpg" alt="">
								</div>
								<p><b>Figure 9.</b> Bar graph to compare relative HMA levels produced by different
									strains. Solid black dots indicate the presence of the
									plasmids, hollow black dots indicate their absence.
								</p>
								<p>Fermentation was carried out for a long time using the HMA5 and HMA6 clones.
									Aliquots
									were collected every two hours,
									compared with HmaS<sup>WT</sup>, the HmaS<sup>V152G</sup> obtained through the
									screening, showed a
									4.73-fold increase in HMA production and
									produced 3.63 g/L HMA after 24 h with a single supply of 20 g/L glucose
									(Fig.
									10).
								</p>
								<div class="figure-container" style="width: 60%;margin: 0 auto;">
									<img src="../../images/20.png" alt="">
								</div>
								<p><b>Figure 10.</b>Comparison of HMA production by the HMA5 and HMA6 strains. The
									bacteria were fermented in shaking flasks for 24 h with a
									single supply of 20 g/L glucose as the substrate, and HMA levels monitored at
									indicated time points</p>
								</p>
							</div>
						</div>
					</div>
			</div>
			</section>
		</div>
	</div>
	</div>
	<a href="#" id="mobile-to-top-arrow" class="to-top">
		<p style="font-size: 20px;">TOP</p>
		<img style="max-height: 60px" src="../../static/home/top.png">
	</a>
	<div class="Container">
		<!--Waves Container-->
		<waves style="margin-bottom:10%">
			<svg class="waves" viewBox="0 24 150 28" preserveAspectRatio="none" shape-rendering="auto">
				<defs>
					<path id="gentle-wave"
						d="M-160 44c30 0 58-18 88-18s 58 18 88 18 58-18 88-18 58 18 88 18 v44h-352z" />
				</defs>
				<g class="parallax">
					<use xlink:href="#gentle-wave" x="48" y="0" fill="rgba(255,192,203,0.7)" />
					<use xlink:href="#gentle-wave" x="48" y="3" fill="rgba(255,255,255,0.5)" />
					<use xlink:href="#gentle-wave" x="48" y="5" fill="rgba(255,255,255,0.3)" />
					<use xlink:href="#gentle-wave" x="48" y="7" fill="rgba(255,255,255,0.1)" />
				</g>
			</svg>
			<!--Waves end-->
	</div>
	<!---------- 页脚FOOTER ------------>
	<footer>
		<div>
			<div class="c-footer">
				<div class="u-container">
					<div class="u-hide-print">
						<div class="c-footer__header">
						</div>
						<div class="c-footer__grid">
							<div class="c-footer__group">
								<h3 class="c-footer__heading">About us</h3>
								<ul class="c-footer__list">
									<li class="c-footer__item"><a href="#" data-track="click">
											This team is composed of enthusiastic students
											from three different colleges and various majors. To spy out the secrets
											of directed evolution, a group of beginners are looking forward to
											creating extraordinary experiences and we did it.</a></li>
								</ul>
							</div>
							<div class="c-footer__group">
								<h3 class="c-footer__heading">About topic</h3>
								<ul class="c-footer__list">
									<li class="c-footer__item"><a href="#" data-track="click"> A dual selection
											system for directed evolution to identify allosteric transcription
											factor PobR variants responsive to different aromatic compounds
										</a></li>
								</ul>
							</div>
							<div class="c-footer__group">
								<h3 class="c-footer__heading">Quicklinks</h3>
								<ul class="c-footer__list">
									<li class="c-footer__item"><a href="../project/background.html"
											data-track="click">Background</a></li>
									<li class="c-footer__item"><a href="../project/design.html"
											data-track="click">Design</a></li>
									<li class="c-footer__item"><a href="../project/results.html"
											data-track="click">Result</a></li>
								</ul>
							</div>
							<div class="c-footer__group">
								<h3 class="c-footer__heading">Contact info</h3>
								<ul class="c-footer__list">
									<li class="c-footer__item"><a href="#" data-track="click">
											Email : iDEC_NEFU_2022@163.com</a></li>
									<li class="c-footer__item"><a href="#" data-track="click">
											Address : Northeast
											Forestry University, Harbin,
											Heilongjiang 150040, P. R. China</a></li>
								</ul>
							</div>
						</div>
					</div>
				</div>
			</div>
		</div>
		<div class="c-corporate-footer">
			<div class="u-container">
				<img src="../../static/logo.png" width="120px" height="40px">
				<p class="c-corporate-footer__legal">iDEC-2023 NEFU_China</p>
			</div>
		</div>
	</footer>

	<script src="../../js/jQuery.js" type=text/javascript></script>
	<script src="../../js/detect.js" type="text/javascript"></script>
	<script src="../../js/main_javascript.js" type=text/javascript></script>
	<script src="../../js/MoveContent.js" type="text/javascript"></script>
	<script src="https://kit.fontawesome.com/50c1a3d91f.js" crossorigin="anonymous"></script>

	<script>
		document.addEventListener("DOMContentLoaded", function (event) {
			MoveContentOutside();
		});
	</script>
</body>

</html>